test/e2e/network/service.go

```diff
@@ -2886,29 +2885,29 @@ var _ = common.SIGDescribe("SCTP [Feature:SCTP] [LinuxOnly]", func() {
 		cs = f.ClientSet
 	})
 
-	ginkgo.It("should allow creating a basic SCTP service with pod and endpoints", func() {
-		serviceName := "sctp-endpoint-test"
+	ginkgo.It("should allow creating Pods, Services and NetworkPolicies with SCTP ports", func() {
```

Comment:
this PR does not actually add any NetworkPolicy test

Reply:
I wasn’t quite sure if that was happening or not but I then just copied the test name from what you had in
> The three tests should be replaced with a single simpler test, something like "should allow creating Pods, Services, and NetworkPolicies with SCTP ports". (It probably doesn't belong in service.go any more at this point though.) It doesn't try to use the Pod/Service/NetworkPolicy it creates, it just confirms that (a) you can create them, and (b) the result of CheckSCTPModuleLoadedOnNodes doesn't change from the start of the test to the end of the test

If the `NetworkPolicies` part of the test needs to be brought in from somewhere or written fresh let me know